src/main/java/eu/dissco/dataexporter/repository/DataExporterRepository.java

```diff
@@ -70,13 +85,54 @@ public String getUserEmailFromJobId(UUID id) {
         .fetchOne(EXPORT_QUEUE.DESTINATION_EMAIL);
   }
 
-  private JSONB mapToJSONB(JsonNode params) throws InvalidRequestException {
+  public Optional<ExportJob> getNextJobInQueue() {
+    var result = context.select(EXPORT_QUEUE.asterisk())
+        .from(EXPORT_QUEUE)
+        .where(EXPORT_QUEUE.TIME_SCHEDULED.eq(
+            context.select(min(EXPORT_QUEUE.TIME_SCHEDULED))
```

Comment:
good point, I was thinking sorting might be more expensive than an inner query but if we only have a few scheduled jobs at any given time it would be better to sort